lib/api/core/plugins/pluginsManager.js

```diff
@@ -1106,15 +1106,17 @@ class PluginsManager {
           plugin.manifest.name);
       }
 
-      if (loadedPlugins[plugin.manifest.name]) {
+      const key = plugin.manifest.name.toLowerCase();
+
+      if (loadedPlugins[key]) {
         errorsManager.throw(
           'plugins',
           'runtime',
           'plugin_name_already_exists',
-          plugin.manifest.name);
+          key);
       }
 
-      loadedPlugins[plugin.manifest.name] = plugin;
+      loadedPlugins[key] = plugin;
```

Comment:
Problem is, we're still building the PluginContext object using the manifest name, and that means that if a plugin containing uppercases in its name tries to create a storage space in ES, it will receive an exception, because ES does not allow the creation of indexes with uppercase letters.